layer/event_logging.h

```diff
@@ -89,6 +89,7 @@ using VectorInt64Attr =
 using StringAttr = AttributeImpl<std::string, ValueType::kString>;
 using Int64Attr = AttributeImpl<int64_t, ValueType::kInt64>;
 using BoolAttr = AttributeImpl<bool, ValueType::kBool>;
+using DurationNsAttr = AttributeImpl<int64_t, ValueType::kInt64>;
```

Comment:
We would need a separate ValueType id to represent duration. One way would be to create a new type (struct/class) representing durations and adding an id for it in the `ValueType` enum.